### PR TITLE
content changes as per LTD-2411

### PR DESCRIPTION
**Pull request by sekharpanja**
content changes as per LTD-2411

### CODE REVIEW

File: api/audit_trail/formatters.py [hunk 1]
```diff
@@ -5,3 +5,16 @@ def removed_flags(**payload):
     elif len(flags) >= 2:
         formatted_flags = f"{str(flags[:-1])[1:-1]} and '{flags[-1]}'"
         return f"removed the flags {formatted_flags} from the organisation"
+
+
+def licence_status_updated(**payload):
+    status = payload["status"].lower()
+    licence = payload["licence"]
+    if status == "issued":
```
Comment (saruniitr) [inline api/audit_trail/formatters.py:13]: please use available enums for this.

File: api/audit_trail/payload.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def format_payload(audit_type, payload):
     AuditType.GENERATE_CASE_DOCUMENT: strings.Audit.GENERATE_CASE_DOCUMENT,
     AuditType.ADD_CASE_OFFICER_TO_CASE: strings.Audit.ADD_CASE_OFFICER_TO_CASE,
     AuditType.REMOVE_CASE_OFFICER_FROM_CASE: strings.Audit.REMOVE_CASE_OFFICER_FROM_CASE,
-    AuditType.GRANTED_APPLICATION: strings.Audit.GRANTED_APPLICATION,
+    AuditType.GRANTED_APPLICATION: "issued licence for {licence_duration} months starting from {start_date}",
```
Comment (currycoder) [inline api/audit_trail/payload.py:107]: I believe this is missing a comma after "months" according to the jira story

Comment (currycoder) [reply api/audit_trail/payload.py:107]: And the [Name] at the start + a full stop on the end..?  Unclear on whether this is added elsewhere..?

**Review by currycoder — COMMENTED** [review]
This looks good although I see a few cases mentioned in the jira that are missing.  Can I assume that these changes would be done in the lite-frontend repo instead?

File: api/audit_trail/payload.py (hunk 2 above)
Comment (sekharpanja) [reply api/audit_trail/payload.py]: This method isn't enough to cover existing cases, so fixed it differently.

File: api/audit_trail/formatters.py [hunk 3]
```diff
@@ -85,3 +87,34 @@ def get_updated_status(**payload):
         return "applied for a licence."
     # Default behavior - same as always
     return format_text(strings.Audit.UPDATED_STATUS, **payload)
+
+
+def licence_status_updated(**payload):
+    status = payload["status"].lower()
+    licence = payload["licence"]
+    if status == LicenceStatus.ISSUED:
```
[the remark this answers is not in the document]
Comment (sekharpanja) [reply api/audit_trail/formatters.py:95]: Good point, I need to clarify what is the best course of action.

[the remark this answers is not in the document]
Comment (saruniitr) [reply api/audit_trail/formatters.py:95]: yes, we need exhaust status as well.

**Review by currycoder — APPROVED** [review]

File: api/audit_trail/formatters.py [hunk 4]
```diff
@@ -106,3 +107,28 @@ def product_reviewed(**payload):
...
+    if status == CaseStatusEnum.WITHDRAWN:
+        return f"withdrew licence {licence}."
+
+    return f"{status} licence {licence}."
+
+def granted_application(**payload):
+    start_date = datetime.strptime(payload["start_date"], "%Y-%m-%d").strftime("%-d %B %Y")
+    licence_duration = int(payload["licence_duration"])
+    if licence_duration > 1:
+        return f"issued licence for {licence_duration} months starting from {start_date}."
```
Comment (saruniitr) [inline api/audit_trail/formatters.py:123]: missing comma after months, below as well

**Review by saruniitr — APPROVED** [review]

**Pull request merged**